drivers/linux/panda.c

```diff
@@ -283,7 +283,7 @@ static void panda_usb_process_can_rx(struct panda_dev_priv *priv_dev,
   //if (msg->dlc & MCBA_DLC_RTR_MASK)
   //  cf->can_id |= CAN_RTR_FLAG;
 
-  cf->can_dlc = get_can_dlc(msg->bus_dat_len & PANDA_DLC_MASK);
+  cf->can_dlc = (min_t(__u8, (msg->bus_dat_len & PANDA_DLC_MASK), 8));
```

Comment:
looks like this got renamed to `can_cc_dlc2len` https://lwn.net/Articles/836683/

Reply:
Will try later

Reply:
This does work and it seems to be reliable, but I'm not familiar with module development, so idk how I would go about differentiating between between different versions of different kernels.

Assistance in this regard would be appreciated.